app/src/main/res/values/strings.xml

```diff
@@ -191,7 +191,7 @@
   <string name="no_nearby">No nearby places found</string>
   <string name="warning">Warning</string>
   <string name="duplicate_image_found">Duplicate Image Found</string>
-  <string name="upload_image_duplicate">This file already destroyed on Commons. Are you sure you want to proceed?</string>
+  <string name="upload_image_duplicate">This file already exists on Commons. Are you sure you want to proceed?</string>
```

Comment:
Hey @maxbetton this change seems irrelevant, correct me if I am wrong. If it has been made by mistake, can you please fix it?

Reply:
reffering to issue #4197 I think i'm right